packages/js/product-editor/src/blocks/generic/select/block.json

```diff
@@ -18,9 +15,6 @@
 		"property": {
 			"type": "string"
 		},
-		"placeholder": {
-			"type": "string"
-		},
```

Comment:
Thanks for removing this, and sorry for not mentioning it earlier, but could you also update the README :) 
https://github.com/woocommerce/woocommerce/blob/4ba72b22585d581369f56173fea69d8593660c38/packages/js/product-editor/src/blocks/generic/select/README.md#L42